### PR TITLE
doc: added new doc `README.md` in folder `doc`.

### PR DESCRIPTION
**Pull request by membphis**
add more doc link to `doc-index`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,11 @@
 - [![Gitter](https://badges.gitter.im/apisix/community.svg)](https://gitter.im/apisix/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 - [![Twitter](https://img.shields.io/twitter/follow/apisixfast.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=apisixfast)
 
-## What's APISIX?
-
 APISIX is a cloud-native microservices API gateway, delivering the ultimate performance, security, open source and scalable platform for all your APIs and microservices.
 
 APISIX is based on OpenResty and etcd. Compared with traditional API gateways, APISIX has dynamic routing and plug-in hot loading, which is especially suitable for API management under micro-service system.
 
+[Install](#Install) | [Documentation](doc/doc-index.md) | [Development ENV](#development-manual-of-apisix) | [FAQ](FAQ.md)
```
Comment (moonming) [inline README.md]: better add `README` of doc, not only index for plugin.

File: doc/doc-index-cn.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (moonming) [inline doc/doc-index-cn.md]: `doc-index-cn.md` to `README.md`

File: doc/plugins.md [hunk 3]
```diff
@@ -1,24 +1,6 @@
...
-* [limit-conn](plugins/limit-conn.md): limite request concurrency (or concurrent connections).
-* [prometheus](plugins/prometheus.md): expose metrics related to APISIX and proxied upstream services in Prometheus exposition format, which can be scraped by a Prometheus Server.
-* [OpenTracing](plugins/zipkin.md): support Zikpin and Apache SkyWalking.
-* [grpc-transcode](plugins/grpc-transcode-cn.md): REST <--> gRPC transcoding。
-* [serverless](plugins/serverless-cn.md)：allow to dynamically run Lua code at *different* phase in APISIX.
-* [ip-restriction](plugins/ip-restriction.md): IP whitelist/blacklist.
-* openid-connect
-
-## update plugins without reload service
+## hot reload without reload service
```
Comment (juzhiyuan) [inline doc/plugins.md]: `Hot reload` is fine :) We don't need more description like `without reload service`.

https://flutter.dev/docs/development/tools/hot-reload

File: README.md [hunk 4]
```diff
@@ -8,12 +8,11 @@
 - [![Gitter](https://badges.gitter.im/apisix/community.svg)](https://gitter.im/apisix/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 - [![Twitter](https://img.shields.io/twitter/follow/apisixfast.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=apisixfast)
 
-## What's APISIX?
-
 APISIX is a cloud-native microservices API gateway, delivering the ultimate performance, security, open source and scalable platform for all your APIs and microservices.
 
 APISIX is based on OpenResty and etcd. Compared with traditional API gateways, APISIX has dynamic routing and plug-in hot loading, which is especially suitable for API management under micro-service system.
 
+[Install](#Install) | [Documentation](doc/README.md) | [Development ENV](#development-manual-of-apisix) | [FAQ](FAQ.md)
```
Comment (juzhiyuan) [inline README.md]: it should be `Installation`

**Pull request merged**